autopilot-daemon/pkg/utils/functions.go

```diff
@@ -144,4 +156,51 @@ func CreateJob(healthcheck string) {
 	if err != nil {
 		klog.Info("Couldn't create Job ", err.Error())
 	}
+	klog.Info("Created")
+}
+
+func CreatePVC() error {
+	cset := GetClientsetInstance()
+	storageclass := os.Getenv("STORAGE_CLASS")
+	pvcTemplate := corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: os.Getenv("POD_NAME"),
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			StorageClassName: &storageclass,
+			AccessModes: []corev1.PersistentVolumeAccessMode{
+				corev1.ReadWriteMany,
+			},
+			Resources: corev1.VolumeResourceRequirements{
+				Requests: corev1.ResourceList{
+					"storage": resource.MustParse("100Mi"),
```

Comment:
Would changing this size matter to the test, maybe based on the storage class being used? If so, we can make it a env var, like `PVC_TEST_STORAGE_SIZE`

Reply:
Good point. Actually, that size is not even true because Scale is configured to have a minimum of 1Gi.. But also I don't think that the amount of disk has any meaning. We only need to know that pvcs can be created